storage/innobase/trx/trx0rseg.cc

```diff
@@ -201,7 +202,7 @@ bool trx_rseg_read_wsrep_checkpoint(const buf_block_t *rseg_header, XID &xid)
 	memcpy(xid.data, TRX_RSEG + TRX_RSEG_WSREP_XID_DATA
 	       + rseg_header->page.frame, XIDDATASIZE);
 
-	return true;
+	return (wsrep_is_wsrep_xid(&xid));
```

Comment:
Please do not add redundant parentheses after the `return` keyword.

Reply:
ok